test/e2e/testing-manifests/kubectl/redis-master-pod.yaml

```diff
@@ -1,33 +0,0 @@
-apiVersion: v1
```

Comment:
test-cmd was using this file. failures were masked by https://github.com/kubernetes/kubernetes/commit/313044abd7a23699b5d744c4157ca7392673d587#diff-7976a1974df5995ca28f32cc945e92acL135. fixing in https://github.com/kubernetes/kubernetes/pull/84871